mmv1/products/compute/terraform.yaml

```diff
@@ -3221,6 +3221,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_value: :NONE
       proxyBind: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  RegionTargetTcpProxy: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "region_target_tcp_proxy_basic"
+        primary_resource_id: "default"
+        vars:
+          region_target_tcp_proxy_name: "test-proxy"
+          region_backend_service_name: "backend-service"
+          health_check_name: "health-check"
+    properties:
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: proxyId
+      service: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: backendService
+      proxyHeader: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_value: :NONE
```

Comment:
Default value should go in api.yaml if possible

Reply:
Should I also fix this for `google_compute_region_target_http_proxy` and `google_compute_region_target_https_proxy`?